main.tf

```
@@ -247,6 +247,10 @@ resource "helm_release" "nginx_ingress_controller" {
   create_namespace = true
   repository       = "https://kubernetes.github.io/ingress-nginx"
   chart            = "ingress-nginx"
+  version          = "3.31.0"
+  values = [
+    "${file("${path.module}/helm/nginx-ingress-values.yaml")}"
```

For reference so that we know where to drag this file from next time we want to update this.

```suggestion
    # values.yaml contents copied from official repo at https://github.com/kubernetes/ingress-nginx/releases/tag/helm-chart-3.31.0
    "${file("${path.module}/helm/nginx-ingress-values.yaml")}"
```